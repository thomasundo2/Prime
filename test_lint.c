int main(){
    mpz_t test; //lint test;
    mpz_init(test);
    //every time run across storage and run contructor mpz_init()
    
    mpz_set_str(test, "123412341341231324132132412413241234123", 10); //test = "123412341341231324132132412413241234123";
    mpz_out_str(stdout, 10, test); //printl(test);
    {
    mpz_t tmp_sum; 
    mpz_init(tmp_sum);
    mpz_add(tmp_sum, test, test); //test + test;
    mpz_out_str(sstdout, 10, tmp_sum); //printl(test + test);
    mpz_clear(tmp_sum);
    }
    
    mpz_clear(test);

    return 0;
}
