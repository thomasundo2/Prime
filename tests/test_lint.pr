int main()
{
    lint test;
    test = "123412341341231324132132412413241234123";
    printl(test);
    printl(test + test);
    /* printl(test ^ 2); */
    return 0;
}