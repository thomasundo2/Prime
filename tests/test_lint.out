123412341341231324132132412413241234123
246824682682462648264264824826482468246
15230605995324594183791325244759532281324681198513403879825706931500099579129